api/lib/src/lib.rs

```diff
@@ -549,6 +549,16 @@ pub trait SimpleSubmissionApi: SignedExtrinsicApi {
 #[async_trait]
 impl<T: SignedExtrinsicApi + Sized + Send + Sync + 'static> SimpleSubmissionApi for T {}
 
+pub type TransactionInIdFor<C> = <<C as Chain>::ChainCrypto as ChainCrypto>::TransactionInId;
+
+#[derive(Serialize, Deserialize)]
+pub enum TransactionInId {
+	Bitcoin(TransactionInIdFor<cf_chains::Bitcoin>),
+	Ethereum(TransactionInIdFor<cf_chains::Ethereum>),
+	Arbitrum(TransactionInIdFor<cf_chains::Arbitrum>),
+	// other variants reserved for other chains.
+}
+
```

Comment:
I assume this should be imported from `cf-rpc-types`?

Reply:
Actually I think the Arbitrum BLS commit wasn't included here at all...